tests/App/AppKernel.php

```diff
@@ -71,6 +71,9 @@ public function getProjectDir()
         return __DIR__;
     }
 
+    /**
+     * @psalm-suppress DeprecatedClass use RoutingConfigurator when dropping support for symfony/routing < 5.1.
```

Comment:
This can be fixed like it was done in SonataMediaBundle

Reply:
Done